Core/Core/Inbox/ComposeMessage/ViewModel/ComposeMessageViewModel.swift

```diff
@@ -20,26 +20,25 @@ import Combine
 import CombineExt
 import CombineSchedulers
 
-class ComposeMessageViewModel: ObservableObject {
+final class ComposeMessageViewModel: ObservableObject {
     // MARK: - Outputs
     @Published public private(set) var recipients: [Recipient] = []
-    @Published public private(set) var isSendingMessage: Bool = false
-
     @Published public var isFilePickerVisible: Bool = false
     @Published public var isImagePickerVisible: Bool = false
     @Published public var isTakePhotoVisible: Bool = false
     @Published public var isAudioRecordVisible: Bool = false
+    @Published public private(set) var showLoader: Bool = false
```

Comment:
For consistency please use `is` for Bool variables where applicable (or `has`, `can`, `should` etc)
In this case `isLoaderVisible` seems okay.